example/caddy_global_options

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Can you please explain why do we need this file? 🤔

Reply:
If someone is playing around with the example, `myapp.com` does not exist as a valid endpoint for the service in the container. So to avoid having the user run up against rate-limits for the production Let's Encrypt endpoint, I'm setting it to the staging environment. Check out https://caddyserver.com/docs/automatic-https#testing